driver/src/main.rs

```diff
@@ -73,6 +73,11 @@ struct Options {
     #[structopt(long, env = "SOLVER_TYPE", default_value = "naive-solver")]
     solver_type: SolverType,
 
+    /// Which optimization solver the solver should use internally. Is passed as
+    /// `--solver` to the solver.
+    #[structopt(long, env = "SOLVER_INTERNAL_SOLVER")]
```

Comment:
naming nit: the double solver reads a bit weird. How about calling it SOLVER_INTERNAL_OPTIMIZER ?

Reply:
It doesn't matter whether it reads weirdly, just that the name makes sense but I am fine with that name to so I'll change it.